spec/factories/accounts.rb

```diff
@@ -0,0 +1,14 @@
+FactoryGirl.define do
+  trait :with_account_owner do
```

Comment:
Interesting having a factory with only a trait. It works like I'd hoped in that you cannot create generic `Accounts` but the trait is there in the subclasses. Neat.